2023-11-29-diving-into-jtag-part-2.md

```diff
@@ -0,0 +1,218 @@
+---
+
+title: Diving into JTAG protocol. Part 2 — Debugging.
+description:
+    "This article explores debugging microcontrollers using JTAG, with a special focus on the ARM Cortex-M architecture and the STM32F407VG microcontroller."
+author: Aliaksandr Kavalchuk
+tags: arm, mcu, debugging, debugger
```

Comment:
Nice! just need to wrap in square brackets (and let's add the cortex-m tag while we're at it!)

```suggestion
tags: [arm, cortex-m, mcu, debugging, debugger]
```

Reply:
Done! thanks for the tip, now I know for the future :)